presto-docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -67,22 +67,39 @@ that catalog name instead of ``sqlserver`` in the above examples.
 SQL Server Connector Limitations
 --------------------------------
 
-Presto supports connecting to SQL Server 2016, SQL Server 2014, SQL Server 2012
-and Azure SQL Database.
+Presto supports connecting to SQL Server 2016, SQL Server 2014, SQL Server 2012,
+SQL Server 2008 R2, SQL Server 2008 and Azure SQL Database.
 
 Presto supports the following SQL Server data types.
 The following table shows the mappings between SQL Server and Presto data types.
 
 ============================= ============================
 SQL Server Type               Presto Type
 ============================= ============================
+``bit``                       ``boolean``
 ``bigint``                    ``bigint``
-``smallint``                  ``smallint``
 ``int``                       ``integer``
+``smallint``                  ``smallint``
+``tinyint``                   ``tinyint``
+``real``                      ``real``
 ``float``                     ``double``
+``decimal``                   ``double``
+``numeric``                   ``double``
 ``char(n)``                   ``char(n)``
+``nchar(n)``                  ``char(n)``
 ``varchar(n)``                ``varchar(n)``
+``nvarchar(n)``               ``varchar(n)``
```

Comment:
That's the thing, if you specify `varchar` without length in T-SQL it's equivalent to `varchar(8000)` in Presto whilst `varchar(max)` is the unbounded (2GB) `varchar` in Presto, same goes for `char`. Maybe I should indeed add the `varchar(max)` = `varchar` lines.